apps/seqan_flexbar/seqan_flexlib.cpp

```diff
@@ -257,11 +271,17 @@ void ArgumentParserBuilder::addAdapterTrimmingOptions(seqan::ArgumentParser & pa
         "np", "no-paired", "Trim paired-end input with single-end trimming method.");
     addOption(parser, pairedModeOpt);
 
-    seqan::ArgParseOption rateOpt = seqan::ArgParseOption(
+    seqan::ArgParseOption errorOpt = seqan::ArgParseOption(
```

Comment:
Ah you added an error rate parameter and renamed the old parameter as (absolute) errors :+1: